test/e2e/framework/metrics/metrics_grabber.go

```diff
@@ -383,6 +393,14 @@ func (g *Grabber) Grab(ctx context.Context) (Collection, error) {
 			}
 		}
 	}
+	if g.grabFromAPIServer {
```

Comment:
the same if condition as on line 340. Expected?

Reply:
ah yeah, we should move this block up to 340

Reply:
Moved to the single if block on line 340. But some checks are currently failing..